src/Microsoft.IdentityModel.Tokens/Microsoft.IdentityModel.Tokens.csproj

```diff
@@ -29,6 +29,10 @@
     <DebugSymbols>true</DebugSymbols>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(Configuration)|$(TargetFramework)|$(Platform)'=='Debug|net45|AnyCPU' OR '$(Configuration)|$(TargetFramework)|$(Platform)'=='Release|net45|AnyCPU' OR '$(Configuration)|$(TargetFramework)|$(Platform)'=='Release|netstandard2.0|AnyCPU' OR '$(Configuration)|$(TargetFramework)|$(Platform)'=='Release|net472|AnyCPU' OR '$(Configuration)|$(TargetFramework)|$(Platform)'=='Debug|net461|AnyCPU' OR '$(Configuration)|$(TargetFramework)|$(Platform)'=='Release|net461|AnyCPU' OR '$(Configuration)|$(TargetFramework)|$(Platform)'=='Debug|net472|AnyCPU' OR '$(Configuration)|$(TargetFramework)|$(Platform)'=='Debug|netstandard2.0|AnyCPU'">
```

Comment:
Still looks a bit complex. Would it be enough only to filter on TargetFramework like here
https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/blob/db230a10bbbfa49e5c5a994b865d2b3afdaa6fc9/src/Microsoft.IdentityModel.Tokens/Microsoft.IdentityModel.Tokens.csproj#L14

Reply:
I figured the condition is not necessary. I've removed it.

Reply:
It's still here :)